packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -121,7 +121,7 @@ export type TermsOfUse = z.infer<typeof termsOfUseGuard>;
 
 export enum Language {
   English = 'en',
-  Chinese = 'zh-cn',
+  Chinese = 'zh-CN',
```

Comment:
it will be better to move this enum to `phrases` and align usage across the repo

Reply:
If we move `language` to `phrases`, `schemas` will depend on `phrases`.
Is it acceptable?

Reply:
Refactored. PTAL